dist/api_data.py

```diff
@@ -741,6 +741,16 @@ def __ge__(self, other):
             # !!! Must match WT_HS_FILE_MIN
             min='0')
         ]),
+    Config('heuristic_controls', '', r'''
```

Comment:
This new category came from a different PR, right? I thought this was already a thing that existed.

Reply:
That PR was reverted, so I added it back with this PR.